benefits/core/migrations/local_fixtures.json

```diff
@@ -63,8 +59,6 @@
       "client_name": "calfresh-benefits-oauth-client-name",
       "client_id_secret_name": "claims-provider-client-id",
       "authority": "https://example.com",
-      "scope": "verify:calfresh",
-      "claim": "calfresh",
       "scheme": "dev-cal-itp_benefits"
     }
   },
```

Comment:
Now these `claimsprovider` fixtures are all the same -- we only need one of them. Make sure to update the references in the `eligibilityverifier` fixtures below.

Reply:
Ah, that's true. Updated in d6c6d5e0f330cad52127ebe745b029847ad2907d